cmd/cloud/server.go

```diff
@@ -246,7 +250,7 @@ var serverCmd = &cobra.Command{
 		}
 		if installationSupervisor {
 			scheduling := supervisor.NewInstallationSupervisorSchedulingOptions(balancedInstallationScheduling, clusterResourceThreshold, clusterResourceThresholdScaleValue)
-			multiDoer = append(multiDoer, supervisor.NewInstallationSupervisor(sqlStore, kopsProvisioner, awsClient, instanceID, keepDatabaseData, keepFilestoreData, scheduling, resourceUtil, logger, cloudMetrics))
+			multiDoer = append(multiDoer, supervisor.NewInstallationSupervisor(sqlStore, kopsProvisioner, awsClient, instanceID, keepDatabaseData, keepFilestoreData, scheduling, resourceUtil, logger, cloudMetrics, forceCRUpgrade))
```

Comment:
This isn't a comment about this PR, but in general we might want to think about moving all these positional arguments to a struct, lol, 11 positional arguments to `NewInstallationSupervisor` is cumbersome to say the least. It didn't start out this bad but we've been slowly adding arguments to that function and it's getting a bit out of control lol

Reply:
Yeah, that is fair point.